runanalysistask/RunAnalysisCheckHFProd.cc

```diff
@@ -100,17 +118,32 @@ void RunAnalysisCheckHFProd(TString configfilename, TString runMode = "full", bo
 
     if (local)
     {
+        string treeName = "";
+        string treeFriendName = "";
+        string dataFileName = "";
+        if(dataType == "ESD")
+        {
+            treeName = "esdTree";
+            treeFriendName = "esdFriendTree";
+            dataFileName = "AliESDs.root";
+        }
+        else
+        {
+            treeName = "aodTree";
+            treeFriendName = "aodTree";
+            dataFileName = "AliAOD.root";
+        }
+
         // if you want to run locally, we need to define some input
-        TChain *chainESD = new TChain("esdTree");
-        TChain *chainESDfriend = new TChain("esdFriendTree");
+        TChain *chain = new TChain(treeName.data());
+        TChain *chainFriend = new TChain(treeFriendName.data());
 
         // add a few files to the chain (change this so that your local files are added)
-        chainESD->Add(Form("%s/AliESDs.root", pathToLocalESDfiles.data()));
-        chainESDfriend->Add(Form("%s/AliAOD.VertexingHF.root", pathToLocalESDfiles.data()));
```

Comment:
Don't we need still this line in case of ESD is used?

Reply:
It was a typo in the previous version due to a copy paste with our usual run analysis: the `AliAOD.VertexingHF.root` is a deltaAOD file, so is not needed for ESDs. In this case it is also not needed for AODs since we do not have to loop over reco candidates (but only generated particles)